admin/connections.php

```diff
@@ -52,21 +53,47 @@ public function renderer()
                 }
                 ?>
             </div>
-            <form method="post" action="options.php">
+            <form id="wallet_settings_form" method="post" action="options.php">
                 <script>
                     window.addEventListener("DOMContentLoaded", function() {
-                                document.getElementById('load_from_lndconnect').addEventListener('click', function(e) {
-                                        e.preventDefault();
-                                        var lndConnectUrl = prompt('Please enter your lndconnect string (e.g. run: lndconnect --url --port=8080)');
-                                        if (!lndConnectUrl) {
-                                            return;
-                                        }
-                                            var url = new URL(lndConnectUrl);
-                                            document.getElementById('lnp_lnd_address').value = 'https:' + url.pathname;
-                                            document.getElementById('lnp_lnd_macaroon').value = url.searchParams.get('macaroon');
-                                            document.getElementById('lnp_lnd_cert').value = url.searchParams.get('cert');
-                                        });
-                                });
+                        document.getElementById('load_from_lndconnect').addEventListener('click', function(e) {
+                            e.preventDefault();
+                            var lndConnectUrl = prompt('Please enter your lndconnect string (e.g. run: lndconnect --url --port=8080)');
+                            if (!lndConnectUrl) {
+                                return;
+                            }
+                            var url = new URL(lndConnectUrl);
+                            document.getElementById('lnp_lnd_address').value = 'https:' + url.pathname;
+                            document.getElementById('lnp_lnd_macaroon').value = url.searchParams.get('macaroon');
+                            document.getElementById('lnp_lnd_cert').value = url.searchParams.get('cert');
+                        });
+
+                        document.getElementById('lndhub_create_account').addEventListener('click', function(e) {
```

Comment:
I think you are missing a `e.preventDefault()` here?

Reply:
Great catch thanks. just added it